test/unity/toolkits/drawing_classifier/data_utils.hpp

```diff
@@ -0,0 +1,96 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+#ifndef TURI_DRAWING_CLASSIFIER_TESTING_DATA_UTILS_H_
+#define TURI_DRAWING_CLASSIFIER_TESTING_DATA_UTILS_H_
+
+#include <algorithm>
+
+#include <model_server/lib/image_util.hpp>
+#include <toolkits/drawing_classifier/dc_data_iterator.hpp>
+
+namespace turi {
+namespace drawing_classifier {
+
+constexpr size_t IMAGE_WIDTH = 28;
+constexpr size_t IMAGE_HEIGHT = 28;
+
+class drawing_data_generator {
+ 
+ public:
+  /** Builds an SFrame with columns "test_image" and "test_targets".
+   *  Creates an SFrame with num_rows number of rows where each row has 
+   *  a drawing (a grayscale 28x28 image)
+   *  and a corresponding target, which is the row index modulo 
+   *  unique_labels.size().
+   */
+  drawing_data_generator(size_t num_rows,
+      const std::vector<std::string> &unique_labels)
+      : num_rows_(num_rows),
+        unique_labels_(unique_labels) {
+
+    flex_list images(num_rows_);
+    flex_list labels(num_rows_);
+    
+    std::vector<unsigned char> buffer(IMAGE_WIDTH * IMAGE_HEIGHT * 1);
+    for (size_t i = 0; i < num_rows_; ++i) {
+
+      // Each pixel has pixel value equal to the row index (modulo 256).
+      std::fill(buffer.begin(), buffer.end(),
+                static_cast<unsigned char>(1 % 256));
+      images[i] = flex_image(reinterpret_cast<char*>(buffer.data()),
+                             IMAGE_HEIGHT, IMAGE_WIDTH, 1, buffer.size(),
+                             IMAGE_TYPE_CURRENT_VERSION,
+                             static_cast<int>(Format::RAW_ARRAY));
```

Comment:
enum can be converted to int implicitly.

Reply:
I'll try it, thanks.